File: win_scripts/wsh.jscript/m10.function.js
// create function
function capitalize(string)
{
  return string.toUpperCase(); // return fully upppercase string
}

WScript.Echo("The current string is: \"ibm\".")
// call the function
result = capitalize("ibm")

// output results
WScript.Echo("The capitalized string is: \"" + result + "\".");

File: compiled_lang/rust/src/m10.function.rs
fn capitalize(s: &str) -> String {
    s.to_uppercase()
}

fn main() {
    let s = "ibm";
    println!("The current string is: \"{s}\".");

    let result = capitalize(s);
    println!("The capitalized string is: \"{result}\".");
}
